packages/app/lib/devMenuItems.ts

```diff
@@ -9,27 +11,77 @@ if (__DEV__) {
   metroBundlerURL = scriptURL.split('index.bundle?')[0] ?? null;
 }
 
-const devMenuItems = [
-  {
-    name: 'Delete local database',
-    callback: () => purgeDb(),
-  },
+type ExpoDevMenuItem = Parameters<typeof registerDevMenuItems>[0][0];
+
+const simulatorOnlyMenuItems: ExpoDevMenuItem[] = [
   {
-    name: 'Drizzle studio (simulator only)',
+    name: 'Drizzle studio',
     callback: async () => {
       const path = getDbPath() ?? '';
       sendBundlerRequest('open-sqlite', { path });
     },
   },
   {
-    name: 'Dump sqlite database',
+    name: 'Dump SQLite',
     callback: async () => {
-      const path = getDbPath() ?? '';
-      sendBundlerRequest('dump-sqlite', { path });
+      const outputPath = process.env.SQLITE_DUMP_PATH ?? 'dump.sqlite3';
+      const databaseSourcePath = getDbPath() ?? '';
+
+      sendBundlerRequest('dump-sqlite', { databaseSourcePath, outputPath });
+    },
+  },
+  {
+    name: 'Restore SQLite',
+    callback: async () => {
+      const sourcePath = process.env.SQLITE_RESTORE_PATH ?? 'restore.sqlite3';
+      const localDatabasePath = getDbPath();
+      if (localDatabasePath == null) {
+        Alert.alert('Could not find database path');
+        return;
+      }
+
+      const overwriteDatabase = async () => {
+        await sendBundlerRequest('restore-sqlite', {
+          sourcePath,
+          targetPath: localDatabasePath,
+        });
+      };
+
+      Alert.alert(
+        'Overwrite local database?',
+        `This will overwrite your local database with the file on the Metro host at ${sourcePath}.`,
+        [
+          {
+            text: 'Overwrite and restart app',
+            onPress: async () => {
+              await overwriteDatabase();
+              DevSettings.reload('Restoring SQLite database');
+            },
+          },
+          {
+            text: 'Overwrite and reset queries (faster, may cause issues)',
+            onPress: async () => {
+              await overwriteDatabase();
+              api.queryClient.resetQueries();
+            },
+          },
+          { text: 'Cancel', style: 'cancel' },
+        ]
+      );
     },
   },
 ];
 
+const devMenuItems: Promise<ExpoDevMenuItem[]> = DeviceInfo.isEmulator().then(
```

Comment:
there is a synchronous form of `isEmulator`, but I chose async to avoid possibly blocking initial app load